File: Codeforces/Archive/136A.scala
object A136 {
	def main(args: Array[String]) = {
		val params = 
		val n = readInt
		val a = List(readLine.split(" ") map Integer.parseInt)
		var res: List[Int] = Nil
		for (i <- 0 to n - 1)
			res = res(:a(i)) :: List(i) :: res[a(i) + 1:]
		res.foreach(i => print(i + " "))
	}
}

File: Codeforces/Archive/136A.py
#raw_input for python 2.6+ O_o
n = int(raw_input())
a = list(map(int, raw_input().split()))
res = [0] * n
for i in range(0, n):
    res[a[i] - 1] = i + 1
for i in res:
    print(str(i) + " ")